press_kit.md

```diff
@@ -0,0 +1,172 @@
+﻿
+# Press Kit
+
+## Index:
+
+1. [AsyncAPI](#asyncapi)
+
+1.1. [Open governance model](#open-governance-model) 

```

Comment:
This is not going to work as-is. You need to add 4 spaces and then just start another numbered list:

```md
1. [AsyncAPI](#asyncapi)
    1. [Open governance model](#open-governance-model) 
```

Also, don't add empty lines in between or it will not be understood as a list.